src/org/labkey/response/ResponseController.java

```diff
@@ -977,7 +974,7 @@ public void setActivityVersion(String activityVersion)
         @Override
         protected HttpView getTabView(ForwardingSettingsForm form, boolean reshow, BindException errors)
         {
-            return new JspView<>("/org/labkey/response/view/forwarderSettings.jsp", form, errors);
+            return new JspView<>("/org/labkey/response/view/MyStudiesResponseServerSettings.jsp", form, errors);
```

Comment:
JSP files usually start with lowercase (by convention). Not worth changing, just FYI.